pages/dev/intro.md

```diff
@@ -4,11 +4,31 @@ Axelar enables secure, any-to-any communication across blockchains, regardless o
 
 Cross-chain apps built on Axelar are truly permissionless, meaning that their transactions cannot be censored by any oracle, relayer or validator. Axelar introduces no new trust assumptions between proof-of-stake chains.
 
-## Basic functions
+## Basic functionality
 
 Here are two basic cross-chain functions you can add to a dApp using Axelar.
 
 1. Token transfers: Send & receive fungible tokens securely from any chain to any chain, including Cosmos-to-EVM and other complex transfers.
 2. General Message Passing: Call any function on any chain from inside a dApp; compose DeFi functions; move NFTs cross-chain; perform cross-chain calls of any kind that sync state securely between dApps on various ecosystems.
 
-The following sections will show you how to set up a local development environment for testing these functions in your dApp, then deploy to testnet and mainnet.
+## Supported Networks
+
+When sending tokes/remote contract calls to any remote network you need to specify a `destinationChain` as a string. Currently supported values for `destinationChain` are:
```

Comment:
Fix typo: "tokes" -> "tokens."

Reply:
It would also be better to keep language consistent here with the paragraphs above.  I.e.: 

When executing token transfers or calling functions on remote chains, you need to specify a `destinationChain` as a string.